sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/credential/package-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+/**
+ * Credential objects used with Azure Blob storage service clients to authorize requests.
```

Comment:
Should this be capitalized like what we do with Azure Communication Services?

```suggestion
 * Credential objects used with Azure Blob Storage Service clients to authorize requests.
```

Reply:
Maybe? Even the official docs page mixes the casing: https://azure.microsoft.com/en-us/services/storage/blobs/

But I think it makes sense for consistency. I will correct.

Reply:
Resolved in d1f274a70